doc/intl/fr/README.md

```diff
@@ -11,6 +11,8 @@ Dogecoin Core [DOGE, Ð]
 
 </div>
 
+IMPORTANT : À partir d'août 2024, la branche master est devenue la branche principale d'intégration et est devenue instable. Veuillez consulter une version taguée avant de compiler des binaires de production
```

Comment:
I would put "taguée" in italic since it is not a real french word.

Reply:
Taguer is indeed a French verb : Définition de taguer ​​​ verbe transitif
au participe passé Murs tagués. Mentionner le nom de (qqn, qqch.) dans un tag (2). Taguer ses amis sur une photo.

Reply:
We can keep it. I would still put it in italic as it's not exactly the same definition.